components/transactions/index.tsx

```diff
@@ -245,7 +251,16 @@ export const Transactions: NextPage<Props> = ({ type, close, setClose, balanceLi
 						onClick={setMaxValue}>
 						Max
 					</button>
+					
 				</div>
+				{/* {!hasApprove && type === BalanceAction.Deposit && (
+						<div style={{display:'flex', alignItems: 'center'}}>
+							<label>Unlimited Approval?</label>
+							<input type={"checkbox"} onChange={() => {setUnlimited(!unlimited)}} style={{
+								marginLeft: '1em'
+							}}/>
+						</div>
+					)} */}
```

Comment:
This block should be removed too if it's no longer used, the code can be recovered from version control if needed.